examples/basic-node/app.js

```diff
@@ -0,0 +1,12 @@
+var dust = require('dustjs-linkedin');
+
+var tmpl = dust.compile("Hello world! Using Dust version {version}!", "hello");
+dust.loadSource(tmpl);
```

Comment:
Cjs is on my roadmap yeah.

On Mon, Mar 23, 2015, 7:03 AM Jimmy Chan <notifications@github.com> wrote:

> In examples/basic-node/app.js
> <https://github.com/linkedin/dustjs/pull/577#discussion_r26938213>:
>
> > @@ -0,0 +1,12 @@
> > +var dust = require('dustjs-linkedin');
> > +
> > +var tmpl = dust.compile("Hello world! Using Dust version {version}!", "hello");
> > +dust.loadSource(tmpl);
>
> you're right. CJS loading would have been nice. There's compileFn and
> renderSource but both of those use loadSource underneath.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linkedin/dustjs/pull/577/files#r26938213>.
>